wallet-new/src/Cardano/Wallet/API/V1/LegacyHandlers/Wallets.hs

```diff
@@ -86,11 +86,10 @@ newWallet genesisConfig NewWallet{..} = do
     let newWalletHandler CreateWallet  = V0.newWalletNoThrow
         newWalletHandler RestoreWallet = V0.restoreWalletFromSeedNoThrow genesisConfig
         (V1 spendingPassword) = fromMaybe (V1 mempty) newwalSpendingPassword
-        (BackupPhrase backupPhrase) = newwalBackupPhrase
     initMeta <- V0.CWalletMeta <$> pure newwalName
                               <*> migrate newwalAssuranceLevel
                               <*> pure 0
-    let walletInit = V0.CWalletInit initMeta (V0.CBackupPhrase backupPhrase)
+    let walletInit = V0.CWalletInit initMeta $ error "TODO: File should be removed"
```

Comment:
Same here.

Reply:
We then get problems since `wallet` and `wallet-new` are using different `Mnemonic` types.
```haskell
/Users/Johannes/IOHK/cardano-sl/wallet-new/src/Cardano/Wallet/API/V1/LegacyHandlers/Wallets.hs:93:64: error:
    • Couldn't match expected type ‘Pos.Util.Mnemonic.Mnemonic 12’
                  with actual type ‘Cardano.Wallet.Kernel.BIP39.Mnemonic 12’
      NB: ‘Cardano.Wallet.Kernel.BIP39.Mnemonic’ is defined at
            src/Cardano/Wallet/Kernel/BIP39.hs:(84,1)-(87,25)
          ‘Pos.Util.Mnemonic.Mnemonic’
            is defined in ‘Pos.Util.Mnemonic’
                in package ‘cardano-sl-wallet-1.3.0’
    • In the first argument of ‘V0.CBackupPhrase’, namely
        ‘backupPhrase’
      In the second argument of ‘V0.CWalletInit’, namely
        ‘(V0.CBackupPhrase backupPhrase)’
      In the expression:
        V0.CWalletInit initMeta (V0.CBackupPhrase backupPhrase)
   |
93 |     let walletInit = V0.CWalletInit initMeta (V0.CBackupPhrase backupPhrase)
   |                                                                ^^^^^^^^^^^^
```

Reply:
Meh. That's exactly why I wanted to remove all those modules first :/ ... 
Fine then, but let's only replace the minimum to make stuff compile and wait for the external handlers to be moved